dashboards-observability/public/components/explorer/explorer.tsx

```diff
@@ -323,11 +323,16 @@ export const Explorer = ({
       }
     }
 
+    if (!isLiveTailOnRef.current) {
+      startTime = curQuery![SELECTED_DATE_RANGE][0];
+      endTime = curQuery![SELECTED_DATE_RANGE][1];
+    }
+
     // compose final query
     const finalQuery = composeFinalQuery(
       curQuery,
-      curQuery![SELECTED_DATE_RANGE][0],
-      curQuery![SELECTED_DATE_RANGE][1],
+      startTime,
+      endTime,
```

Comment:
is it possible for them to be empty strings? i don't think composeFinalQuery handles empty strings

Reply:
ohh i have assigned its value before composeFinalQuery is called